main.tf

```diff
@@ -57,4 +57,7 @@ module "project-factory" {
   disable_dependent_services  = var.disable_dependent_services
   python_interpreter_path     = var.python_interpreter_path
   pip_executable_path         = var.pip_executable_path
+  budget_amount               = var.budget_amount
```

Comment:
I'd actually like to keep this *out* of the core project factory. Instead add a separate module invocation.

Reply:
Moved it out and called budget submodule directly from root and other modules